send/header.c

```diff
@@ -784,6 +786,13 @@ int mutt_write_mime_header(struct Body *a, FILE *fp, struct ConfigSubset *sub)
       struct Parameter *cont = NULL;
       TAILQ_FOREACH(cont, &pl_conts, entries)
       {
+        if (mutt_istr_equal(cont->attribute, "content-id"))
+        {
+          // Content-ID: gets its own header
+          id = mutt_str_dup(cont->value);
```

Comment:
Can the `Parameter` change during this function?
If not, then `id = cont->value;` would do (and we don't need to `FREE()` later)

Reply:
I think I was originally going to add the angle braces in to `id`, but later just used the `fprintf` and forgot I didn't need to make a copy.